src/pages/index.js

```diff
@@ -72,13 +93,15 @@ class CV extends React.Component {
     return (
       <div>
           <CVPage firstPage={true}
```

Comment:
Hier kan je ook zeggen de userinfo doorgeven ipv alle waardes specifiek declareren. En dit doe je ook al goed
                
>   userInfo = { this.state.userInfo}

Gebruik in je CVPage dan userInfo.[de key die je nodig hebt]. Dat maakt het wat cleaner, en zo hoef je niet voor elk veld wat je toevoegt in je userInfo je CVPage te updaten met nieuwe props die je moet declareren.